src/components/Board.js

```diff
@@ -5,29 +5,95 @@ import axios from 'axios';
 import './Board.css';
 import Card from './Card';
 import NewCardForm from './NewCardForm';
-import CARD_DATA from '../data/card-data.json';
+// import CARD_DATA from '../data/card-data.json';
 
 class Board extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
 
     this.state = {
-      cards: [],
+      url: this.props.url,
+      boardName: this.props.boardName,
```

Comment:
If the URL and board name aren't going to be changed within this component, you don't need to keep them in `state`. If you read them from `props` throughout the app, it will be more clear that they're read-only.